target/linux/bcm27xx/bcm2711/config-5.10

```diff
@@ -188,6 +186,7 @@ CONFIG_FONT_8x8=y
 CONFIG_FONT_SUPPORT=y
 CONFIG_FRAMEBUFFER_CONSOLE=y
 # CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY is not set
+# CONFIG_FRAMEBUFFER_CONSOLE_LEGACY_ACCELERATION is not set
```

Comment:
Hm. I'm pretty sure all `CONFIG_FRAMEBUFFER_CONSOLE=y` targets will complain about this missing symbol. Since there are quite a few of them…
```
rui@crystalwell:~/Sources/OpenWrt/openwrt/target$ find . -type f -name "config-5.10" -exec grep CONFIG_FRAMEBUFFER_CONSOLE=y {} +
./linux/bcm27xx/bcm2711/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/bcm27xx/bcm2710/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/bcm27xx/bcm2708/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/bcm27xx/bcm2709/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/sunxi/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/mediatek/mt7623/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/x86/legacy/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/x86/64/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/x86/generic/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/layerscape/armv8_64b/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
./linux/gemini/config-5.10:CONFIG_FRAMEBUFFER_CONSOLE=y
rui@crystalwell:~/Sources/OpenWrt/openwrt/target$ 
```
… it's a good idea to move this symbol to the generic config.